scripts/elasticsearch-test-helpers/destination_connector/test-ingest-elasticsearch-output.py

```diff
@@ -28,3 +56,16 @@
             f"got {count} items in index, expected {N_ELEMENTS} items in index."
         )
     print(f"Elasticsearch destination test was successful with {count} items being uploaded.")
+
+    # Query the index using the appropriate embedding vector for given query text
+    # Verify that the top 1 result matches the expected chunk by checking the start text
+    print("Testing query to the embedded index.")
+    query_text = (
+        "A gathering of Russian nobility and merchants in historic uniforms, "
+        "discussing the Emperor's manifesto with a mix of solemn anticipation "
+        "and everyday concerns, while Pierre, dressed in a tight nobleman's uniform, "
+        "ponders the French Revolution and social contracts amidst the crowd."
+    )
+    query_response = query(client, query_text)
+    assert query_response["hits"]["hits"][0]["_source"]["text"].startswith("CHAPTER XXII")
```

Comment:
query text is designed to semantically match with the beginning text of that chapter